python-runnables/run-on-kubectl/runnable.json

```diff
@@ -2,7 +2,7 @@
     "meta": {
         "label": "Run Kubectl command",
```

Comment:
Maybe a `(deprecated)` after the label here?

Reply:
done